### PR TITLE
Feature/oep query build

### LINKED ISSUE #20: View fetches data about weather point depending on provided mouse-click(lat/lng)
- backend functionality 
- view should be created as class method within the Serializer class
- view returns data about the closest weather point to any provided mouseclick position
- position should be provided as lat/lng value
- fetched information should be provided at an URL 

See #10 for further information on how the mouse-click position is provided

### LINKED ISSUE #29: Figure out how to build a query for the open fred data

### LINKED ISSUE #30: Figure out how to display graphs

### LINKED ISSUE #31: Figure out how to assign popups to points with the correct information
Would be cool to prepare this in the views.py and only use a function in the .js side

### PR DESCRIPTION
**Pull request by Bachibouzouk**
Closes #20, Closes #22, Closes #29, Closes #30, Closes #31

Add a first version of the OEP query through the OEP API

### CODE REVIEW

File: templates/WAM_APP_FRED/test_map_layout.html
Comment (Bachibouzouk) [inline templates/WAM_APP_FRED/test_map_layout.html:295]: this function displays the plot

**Comment by Bachibouzouk** [comment]
@christian-rli - I need your approval before I merge :)

**Review by christian-rli — APPROVED** [review]

**Review by christian-rli — APPROVED** [review]
I highly approve. Eventually we should zoom out all the way to show Germany, I think.

**Comment by Bachibouzouk** [comment]
@jh-RLI - maybe you will have conflict in your branches after I merge this one, make sure to pull develop locally and merge the changes/fix the conflicts locally in your feature branch

**Pull request merged**
